### DISCUSSION TITLE
[Suggestion] Conditional Battle System

### DISCUSSION THREAD

**Discussion by GG-Cannon** [tsunamayo/Starship-EVO#4011] [Suggestions]
I've heard and seen in trailers a few weapons that have been removed from the game to be worked on and that kind of gave me this idea.

Some weapons should work better outside of an atmosfere than they do inside, and some weapons should give more hull damage while others give more shield damage.

It is sort of a rock-paper-scisorish weapons system.

Energy weapons would cause more damage to shields than they do heat damage to hulls, since the hull literally defuses heat.
Further more, energy weapons would be a lot less effective inside an atmosphere, since a lot of the energy is lost to the atmosphere along the way.

Shields would work exactly the same inside an atmosphere than they do outside.

Flaks, Rail-Gatling guns and Rail-Cannons[Kinetics] would do a lot of hull damage while extremely low shield damage.
But they'd be only truly effective while inside atmospheres or on the inside of a ships, having reduced accuracy outside of an atmosphere... since they'd also be limited in ammunition, needing to reload that would mean spending too much ammo while in space would be ill advised.

Flame throwers would be exclusive for inside atmosphere, since they require the atmosphere for combustion.

Missiles would be built things and not just a placed launcher, where you literally have to modularly create the launch system, the reloader and each missile with different possible payloads. Which would mean they are limited in number.
Explosive missiles and EMP missiles.
EMP would cause "energy damage" so it would deplete the ship, turret or base energy depending on how big was the EMP missile.

Further, there'd be a sort of "Cooling system". Could be similar to a combination of a big piece like the shield or big reactor + the "capacitor" for example, in being a module which you can attach to reactors, shields and energy weapons that severely reduce but never really completely voids the heat they produce.
You'd have to build the cooler, a piping system and attach a "capacitor"like piece to the things you want to cool down. They'd never fully cool but the more objects you want to cool down, more coolers and more piping you'd need. And if the cooler gets destroyed, the heat would go up by a lot.

All of these together would create an interesting dynamic.

You'd need to use beams to destroy a shield in space while using missles to cause hull damage. If you destroyed a ship's cooling system or disabled it through EMP, you'd then be able to cause a lot of damage with beam weapons.

While inside atmospheres, you'd have to use EMP missiles to disable shields in order to attack with explosive missiles and ammunition weapons or have truly massive energy weapons capable of causing significant damage even through atmosphere and cooling systems.

This would also make big battle interesting, since small ships would be able to attack carriers from inside their shields but be weaker to damage, being destroyed easily while big ships would be annoyed by those smaller ships but be a lot stronger to other bigger ships and carry several smaller ship inside, but be very strong against bases since they'd have a wider array of weapons and bases would be a lot stronger against smaller ships and have proper defensive measures against bigger ship's beam weapons forcing them to have EMP missiles and not only explosive missiles and beams.

Battles would change in strategy depending on the scenario but Deathstar level ships with massive beam weapons would still be useful, but they'd actually have to have a big cooling system and that would really be a very weak point for small ships to exploit.

**Comment by TIKIRobo** [comment]
Personally I don't think that there should be huge differences in weapons when in or out of atmo just because this game is a bit more arcad-y and so a new player might be confused as to why a certain gun isn't working at times or is using more power or whatever, 
also flamethrowers are sooo 21st century we would ofc be using plasma launchers, but in all seriousness plasma launchers could work in space and atmo so I'd think those would be added instead.
as far as the weapon types doing different damage to shields/hull i do like that, its very similar to how weapons work in Elite

**Comment by willlascas** [comment]
game needs missiles ... and gatling gun can not be missing ... mainly for creating handguns!

**Comment by tsunamayo** [comment]
Sorry I am finishing the neovox update so I havent had time to read this yet... Cheers
